_posts/2016-03-16-Containers-all-the-way-through.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+name: containers-all-the-way-through
+title: Containers all the way through...
+date: 2016-03-16 00:00:00 +00:00
+author: Mani Sarkar
+image:
+    src: /assets/img/custom/blog/2016-03-16-containers-all-the-way-through/cover-image.png
+    attribution:
+        text: World Inside Drops by Joni Niemelä
```

Comment:
this is "All rights reserved" - choose a CC picture please and mention the license